packages/react-hooks/src/useBlockInterval.ts

```diff
@@ -40,10 +42,16 @@ function calcInterval (api: ApiPromise): BN {
 function useBlockIntervalImpl (apiOverride?: ApiPromise | null): BN {
   const { api } = useApi();
 
+  const currApi = apiOverride || api;
+  const blockTimeAura = useCall(currApi.call.auraApi?.slotDuration && currApi.call.auraApi.slotDuration, []);
```

Comment:
```suggestion
  const blockTimeAura = useCall<BN>(currApi.call.auraApi?.slotDuration && currApi.call.auraApi.slotDuration, []);
```

Reply:
Addressed in [0e593af](https://github.com/polkadot-js/apps/pull/10457/commits/0e593afd5d5c90d6b1d2cd36eee770dc647a9138).